src/context/config.ts

```diff
@@ -18,15 +20,23 @@ export enum ConfigFormat {
  * commands require at least some. */
 export interface Config {
   name: string;
-  entry?: string;
+  entry: string;
```

Comment:
Why are we mandating an entry file field? This would introduce a breaking change to anyone who uses the default `mod.ts` and doesn't provide an entry file.

Reply:
The entry file is not mandatory, this is just some typescript syntaxic sugar, let me explain
At the beginning, the `egg` object is of the type `Partial<Config>`.
https://github.com/nestdotland/eggs/blob/revamped-publish-command/src/commands/publish.ts#L241
Since the field can come from the configuration file or from the command option, by default this field is not mandatory.
`entry` is then set to `mod.ts` if nothing is provided. `egg` is now of type `Config`.
https://github.com/nestdotland/eggs/blob/revamped-publish-command/src/commands/publish.ts#L251
What is mandatory is the existence of this file. (it was already the case before, I didn't add anything there)